packages/components/src/circular-option-picker/index.tsx

```diff
@@ -118,3 +130,51 @@ export default function CircularOptionPicker( props ) {
 CircularOptionPicker.Option = Option;
 CircularOptionPicker.ButtonAction = ButtonAction;
 CircularOptionPicker.DropdownLinkAction = DropdownLinkAction;
+
+/**
```

Comment:
In order to be associated to the `CircularOptionPicker` component, this JSDoc comment needs to be moved next to where the component is declared (basically on top of `function CircularOptionPicke....`)

Reply:
🤦 Fixed, thank you.